lady_luck.js

```diff
@@ -104,7 +86,7 @@ function nwodRoll(pool, options) {
 		outcome.errors.push("Cannot roll more than 25 dice");
 	}
 
-	if (outcome.errors.length > 0) {
+	if (outcome.errors.length) {
```

Comment:
Checking if something is greater than zero is more expressive than checking if it's there at all.

Reply:
JS will check if .length is "true" here. JS treats true and 1 as identical for these kind of checks, it's just a tidier shorthand

Reply:
I know what it does. Checking if something is truthy is less expressive than checking if something is empty.